browser/static/js/main_zstack.js

```diff
@@ -382,12 +382,10 @@ class Mode {
       } else if (this.action === 'predict') {
         action('predict_single', { frame: current_frame });
       } else if (this.action === 'replace') {
-        if (this.info.label_1 !== this.info.label_2 &&
-            this.info.frame_1 === this.info.frame_2) {
+        if (this.info.label_1 !== this.info.label_2) {
           action('replace_single', {
             label_1: this.info.label_1,
-            label_2: this.info.label_2,
-            frame: this.info.frame_1
+            label_2: this.info.label_2
```

Comment:
We no longer include `frame` in the action payload because we are replacing it in the "active" frame. Was `frame_1` always considered the "active" frame as well? I just want to make sure we aren't changing any behavior here.